core/src/main/resources/filodb-defaults.conf

```diff
@@ -30,8 +30,7 @@ filodb {
 
     options {
       copyTags = {
-        exporter = "_ns_"
-        job = "_ns_"
+        "_ns_" = [ "_ns", "exporter", "job" ]
```

Comment:
Could we add a comment to explain what this means?   Thanks

Reply:
Added comments.